system-parachains/people-paseo/src/xcm_config.rs

```diff
@@ -324,6 +363,10 @@ impl xcm_executor::Config for XcmConfig {
 			Balances,
 			ResolveTo<StakingPot, Balances>,
 		>,
+		// Allows for paying for XCM execution from funds in XCM register.
+		// Useful for accounts having no funds on this chain but transferring
+		// assets accepted as fees here.
+		FixedRateOfFungible<HollarPerSecondPerMegabyte, HollarToStakingPot>,
```

Comment:
Is this a hack or what we want to have in production? because HollarPerSecondPerMegabyte values seem wrong.

Also this should be handled by the item below. Which takes funds from `AssetsInHolding` register. If `UsingComponents` doesn't work we may want to copy the code from asset-hub IIRC they are using another technique.

But probably it wasn't working just because `StableAssetLocation` was pointing to USDC and not HOLLAR.

Reply:
Before adding this, the transfer from hydration to people lite work was stopped at the UsingComponents traders with TooExpensive (if I remember correctly) error. So it looks like we lacked a trader that would take the funds from AssetsInHolding register.

I didn't put too much attention to the exact values here as this is a testnet and also I wanted to have a discussion on them during the review.